scss/index.scss

```diff
@@ -0,0 +1,134 @@
+@import "./variables.scss";
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+.container {
+  position: relative;
+  max-width: 100%;
+  // top: -62px;
+}
+
+.links {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 15px 80px;
+  z-index: 1000;
+  @media (max-width: $mediumScreen) {
+    padding: 10px 0px;
+  }
+  background-color: #000;
+  li {
+    transition: all 100ms linear;
+    font-family: $smallerFontStyle;
+    font-size: $smallFont;
+    @media (max-width: $mediumScreen) {
+      font-size: $verySmallFont;
+    }
+    &:hover {
+      transition: all 100ms linear;
+      transform: scale(1.3);
+    }
+    a {
+      color: hsl(300, 62%, 100%);
+      text-decoration: none;
+      font-weight: bold;
+    }
+  }
+}
+
+.section {
+  width: 100%;
+  height: 100vh;
+  background-color: hsl(293, 62%, 100%);
+  display: flex;
+}
+
+.intro {
+  display: grid;
+  min-width: 80%;
+  max-width: 80%;
+  height: 100vh;
+  margin: auto;
+  grid-template-columns: 2fr 1fr;
+  @media (max-width: $mediumScreen) {
+    grid-template-columns: 1fr 0fr;
+    min-width: 100%;
+    padding-top: 0px;
+  }
+  .profile {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    h1 {
+      text-align: center;
+      font-family: $biggerFontStyle;
+      font-weight: normal;
+      letter-spacing: 4px;
+      font-size: $veryLargefont;
+      @media (max-width: $mediumScreen) {
+        font-size: $extraLargefont;
+      }
+    }
+    .illustration {
+      display: none;
+      @media (max-width: $mediumScreen) {
+        display: inline-block;
+        width: 500px;
+      }
+    }
+    .aboutBtn {
+      background-color: #000000;
+      padding: 15px;
+      width: 150px;
+      outline: none;
+      border: none;
+      border-radius: 5px;
+      transition: all 100ms linear;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      color: hsl(293, 62%, 100%);
+      font-family: $smallerFontStyle;
+      text-decoration: none;
+      font-size: $smallFont;
+      @media (max-width: $mediumScreen) {
+        font-size: $verySmallFont;
+        padding: 8px;
+        width: 100px;
+      }
+      font-weight: bold;
+      &:hover {
+        transition: all 100ms linear;
```

Comment:
can we specify only the properties you want to transition on instead of `all` for performance reasons something like:
`transition: background-color 100ms linear, color 1s ease`